[Flutter] Non EVM support docs (#134)
reown-com/reown-docs: docs/appkit/flutter/core/options.mdx

### siweConfig:

Used to configure [One-Click Auth + Sign In With Ethereum](./siwe.mdx) feature. 

Will be disabled automatically if you decide to support/include non-EVM blockchains.

### featuresConfig:

Used to configure extra features such [Email and Social Login](../core/email.mdx).

### getBalanceFallback:

> This callback method will be triggered if getting the balance from our blockchain API fails due to unsupported network. You should place here your own getBalance() method if you want.

Reviewer: ```suggestion
This callback method will be triggered if getting the balance from our blockchain API fails due to unsupported network. You may replace this with your own `getBalance()` method if desired.
```
